core/src/com/gaalf/presenter/BaseGamePresenter.java

```diff
@@ -250,23 +246,8 @@ public void dispose(){
 
     public abstract void levelCleared();
 
-    private Entity createShotIndicator(){
-        SpriteComponent spriteComponent = new SpriteComponent();
-        Texture texture = new Texture("arrow.png");
-        spriteComponent.sprite = new Sprite(texture);
-
-        TransformComponent transformComponent = new TransformComponent();
-        transformComponent.pos.set(3, 3);
-        transformComponent.scale.set(0.2f, 0.2f);
-        transformComponent.rotation = 0;
-        transformComponent.visible = false;
-
-        Entity e = new Entity();
-        e.add(spriteComponent);
-        e.add(transformComponent);
-        e.add(new ShotIndicatorComponent());
-        return e;
-
+    private Entity createShotIndicator(PlayerInfo player){
```

Comment:
This function could be removed. It's only used at line 156, and it's not really needed there anymore.